Automattic/pocket-casts-ios
# Download button on player

| 📘 Part of: # |  
|:---:|

Fixes #https://github.com/Automattic/pocket-casts-android/issues/1860

Implements a download shelf action on the player

https://github.com/user-attachments/assets/0ae4a135-b494-4963-80f8-378d0a989e24

## To test

1. Start the app
2. Play an episode
3. Open the full screen player
4. Tap on the more(...) button 
5. Tap on Download
6. Check that the download is started/queued
7. Wait for the download to finish
8. Check that the shelf action is changed to `Remove ... `
9. Tap on Remove
10. Check if file is deleted and system skips to next episode on the queue

## Checklist
